### PR TITLE
feat: add controverse testnet

### PR DESCRIPTION
**Pull request by levino**

### CODE REVIEW

File: devnets/controverse/chain.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  "metadata": {
+    "op_bridge_id": "116",
+    "op_denoms": ["uinit"],
+    "ibc_channels": [
+      {
+        "chain_id": "mahalo-2",
+        "port_id": "transfer",
+        "channel_id": "channel-0",
+        "version": "ics20-1"
+      },
```
Comment (levino) [inline devnets/controverse/chain.json:90]:
```suggestion
      }
```

**Review by AndreasFinke — APPROVED** [review]

File: devnets/controverse/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "miniwasm",
+  "assets": [
+    {
+      "description": "The native token of Initia",
+      "denom_units": [
+        {
+          "denom": "l2/771d639f30fbe45e3fbca954ffbe2fcc26f915f5513c67a4a2d0bc1d635bdefd",
```
Comment (ALPAC-4) [inline devnets/controverse/assetlist.json:9]: the op bridged init denom is `l2/2716295e82bab3b0308dc1e35837fe471657ea07ee71aebb6129739fb0f1278a` the eaiest why to get it is bridged init with op bridge and check balance (ex https://maze-rest-493360ba-cd86-4627-aca3-8b61635eb2bf.ue1-prod.newmetric.xyz/cosmos/bank/v1beta1/balances/init13rqpwycyhepvxgfmzucat5t6v0veftnf684ua6)

Comment (levino) [reply devnets/controverse/assetlist.json:9]: So you are saying:
```suggestion
          "denom": "l2/2716295e82bab3b0308dc1e35837fe471657ea07ee71aebb6129739fb0f1278a",
```
?

Comment (ALPAC-4) [reply devnets/controverse/assetlist.json:9]: right and please also change the one in chain.json

File: devnets/controverse/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "INIT",
+          "exponent": 6
+        }
+      ],
+      "base": "l2/771d639f30fbe45e3fbca954ffbe2fcc26f915f5513c67a4a2d0bc1d635bdefd",
```
Comment (levino) [inline devnets/controverse/assetlist.json:17]:
```suggestion
      "base": "l2/2716295e82bab3b0308dc1e35837fe471657ea07ee71aebb6129739fb0f1278a",
```
@ALPAC-4 also?

File: devnets/controverse/chain.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  "network_type": "devnet",
+  "bech32_prefix": "init",
+  "daemon_name": "minitiad",
+  "node_home": "$HOME/.minitia",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "l2/771d639f30fbe45e3fbca954ffbe2fcc26f915f5513c67a4a2d0bc1d635bdefd",
```
Comment (levino) [inline devnets/controverse/chain.json]:
```suggestion
        "denom": "l2/2716295e82bab3b0308dc1e35837fe471657ea07ee71aebb6129739fb0f1278a",
```
@ALPAC-4 also?

File: devnets/controverse/assetlist.json (hunk 3 above)
Comment (ALPAC-4) [reply devnets/controverse/assetlist.json]: Yes please change this too

**Comment by AndreasFinke** [comment]
already commited these changes @levino

**Pull request merged**